lib/doctest.js

```diff
@@ -445,32 +436,33 @@ ${arrowWrap (source)}
   }
 };
 
-const attempt = thunk => {
+const encase = f => (...args) => {
```

Comment:
I don't get the necessity for this change, since at all call sites it's immediately called with no arguments. Is it setting something up for a following commit?

Reply:
You're right about this setting up for a future commit. I reverted the change, thinking we should stick with `attempt` until we need the flexibility offered by `encase`, but I just realized that *`attempt` was introduced in this pull request*. It makes sense to introduce the more flexible function given that we'll soon need the flexibility.